### DISCUSSION TITLE
Any character for Token pattern

### DISCUSSION THREAD

**Discussion by DrakeProgt** [beholdr/maska#120] [Q&A] [answered]
Hey,

I want to upgrade my maska for Vue from 1.5.2 to 2.1.4. So I made all the changes which were necessary but I have a problem now with the pattern for tokens.

My Code for 1.5.2:
`<input v-mask="{ mask: 'SSS', tokens: { S: { pattern: /./ } } }">`
Here the any character regex works and I could type any random character until the input field has a length of 3.

My changed Code for 2.1.4:
`<input v-maska data-maska="SSS" data-maska-tokens="S:/./">`
The any character doesn´t work.
I tried also the long version for pattern with: `:data-maska-tokens="{ S: { pattern: /./ } }"` but it doesn´t work too.

Can anyone help me to make this works and I can set the any character regex for token pattern in maska 2.1.4?

Thanks.

Regards Drake

**Comment by beholdr** [comment, marked as answer]
When using simple form for custom tokens you should use string format without delimiters, like this:

``` html
<input v-maska data-maska="SSS" data-maska-tokens="S:.">
```

**Reply by DrakeProgt** [reply]
Thanks :)
